ListApp/ListApp.Tests.Base/ListItemEqualityComparer.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using ListApp.Contracts.Models;
+
+namespace ListApp.Tests.Base
+{
+    internal class ListItemEqualityComparer : IEqualityComparer<ListItem>
```

Comment:
The sole purpose of `ListApp.Tests.Base` is to provide this comparer and the extension method. By making them `internal`, this assembly has no public API and everybody using it has to be listed in its `AssemblyInfo.cs` - that makes it pretty useless assembly.